doc/classes/SceneTree.xml

```diff
@@ -238,76 +237,76 @@
 			If [code]false[/code], you need to manually call [method MultiplayerAPI.poll] to process network packets and deliver RPCs. This allows running RPCs in a different loop (e.g. physics, thread, specific time step) and for manual [Mutex] protection when accessing the [MultiplayerAPI] from threads.
 		</member>
 		<member name="paused" type="bool" setter="set_pause" getter="is_paused" default="false">
-			If [code]true[/code], the [SceneTree] is paused. Doing so will have the following behavior:
-			- 2D and 3D physics will be stopped. This includes signals and collision detection.
-			- [method Node._process], [method Node._physics_process] and [method Node._input] will not be called anymore in nodes.
+			If [code]true[/code], the scene tree is considered paused. This causes the following behavior:
+			- 2D and 3D physics will be stopped, as well as collision detection and related signals.
+			- Depending on the [member Node.process_mode], the nodes' [member Node.process_mode] [method Node._process], [method Node._physics_process] and [method Node._input] may not called anymore.
 		</member>
 		<member name="quit_on_go_back" type="bool" setter="set_quit_on_go_back" getter="is_quit_on_go_back" default="true">
 			If [code]true[/code], the application quits automatically on going back (e.g. on Android).
 			To handle 'Go Back' button when this option is disabled, use [constant DisplayServer.WINDOW_EVENT_GO_BACK_REQUEST].
 		</member>
 		<member name="root" type="Window" setter="" getter="get_root">
-			The [SceneTree]'s root [Window].
+			The tree's root [Window]. This is the beginning of everything.
```

Comment:
Maybe elaborate a bit, since this is an important member.
```suggestion
			The tree's root [Window]. Every node that is added as a child of the [member root] will be part of the application. Children may include the [member current_scene] and any autoloads that were registered.
```